app/services/doc_auth/socure/requests/docv_result_request.rb

```diff
@@ -27,6 +27,14 @@ def handle_http_response(http_response)
           )
         end
 
+        def handle_connection_error(exception:)
+          NewRelic::Agent.notice_error(exception)
+          DocAuth::Socure::Responses::DocvResultResponse.new(
+            http_response: nil,
```

Comment:
this causes a 500 error [here](https://github.com/18F/identity-idp/blob/fbf0e5e9c80f4a8495f82d68e1601c7dd0b0ce42/app/services/doc_auth/socure/responses/docv_result_response.rb#L113) because https_response is nil